web/concrete/src/Application/Application.php

```diff
@@ -58,9 +58,9 @@ public function shutdown($options = array())
             }
 
             $logger = new Logger();
-            $r = Request::getInstance();
+            $r = $this['request']->getInstance();
 
-            if (Config::get('concrete.log.queries.log') &&
+            if ($this['config']->get('concrete.log.queries.log') &&
```

Comment:
The only change I'd make is to only use the `\ArrayAccess` methods when you absolutely need to, so instead of:

```php
$this['some_abstract']->doSomething();
$this['some_abstract']->doSomethingElse();
$this['some_abstract']->finishDoingStuff();
```

It's much better to store the result of `$this['some_abstract']` to a variable:

```php
$abstract = $this['some_abstract'];

$abstract->doSomething();
$abstract->doSomethingElse();
$abstract->finishDoingStuff();
```

(in reference to multiple `$this['config']` calls in the same method)